src/sentry/static/sentry/app/components/dropdownBubble.tsx

```diff
@@ -40,7 +37,8 @@ const getMenuBorderRadius = ({
   alignMenu,
   width,
   theme,
-}: Params) => {
+}: // eslint-disable-next-line no-use-before-define
+React.ComponentProps<typeof DropdownBubble>) => {
```

Comment:
@billyvg was able to remove the required theme prop with `React.ComponentProps<typeof DropdownBubble>` its a little ugly with the eslint ignore

Reply:
Hmm this can still be `Params`, yeah?

Reply:
Ohh I see what's going on

Reply:
@scttcper Hmm, in this case I would leave theme in because of these helper functions that return css and that they're all contained in the same module.